pkg/apis/pipeline/v1alpha1/taskrun_types.go

```diff
@@ -136,11 +136,24 @@ type TaskRunStatusFields struct {
 	// optional
 	ResourcesResult []PipelineResourceResult `json:"resourcesResult,omitempty"`
 
+	// TaskRunResult from task runs
```

Comment:
`TaskRunResults` to match the field name.  Maybe this should be something like, "TaskRunResults are the list of results written out by the task's containers"